templates/Procfile

```diff
@@ -1,2 +1,2 @@
-web: bundle exec unicorn -p $PORT -c ./config/unicorn.rb
+web: bundle exec rails serve Puma -p $PORT -c ./config/puma.rb
```

Comment:
I don't think there are differences really. There were some conflicts with some flags being parsed and passed to `rails`. The main reason we're doing it that way is so anyone running straight Rack can copy most of the article verbatim and still have it work. Same command on every project. 

Btw, i think `serve` should be `server`. Also have you considered providing default config values? So you can run without a port specified i.e. `-p ${PORT:-3000}`